Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()
     {
-        Debug.Log(stamina);
-        if(stamina > 0)
-        {
-            stamina--;
-            return true;
-        }
-        else
-        {
-            crash = true;
-            return false;
-        }
+        if (debug)
+            Debug.Log("Standard Restore: ");

```

Comment:
please look at code, all debug messages are and should be put on flaggs, and no I cant remove them, they are needed for aditional development.